pkg/server/warcserver/daterange.go

```diff
@@ -18,25 +18,24 @@ package warcserver
 
 import (
 	"fmt"
-	"strings"
 )
 
-type dateRange struct {
+type DateRange struct {
 	from string
 	to   string
 }
 
-func parseDateRange(from, to string) *dateRange {
-	from = fmt.Sprintf("%s%0*d", from, 14-len(from), 0)
-	to = fmt.Sprintf("%s%.*s", to, 14-len(to), "99999999999999")
-	d := &dateRange{from: from, to: to}
-	return d
+// contains returns true if the timestamp ts contained by the bounds defined by the DateRange d.
+func (d DateRange) contains(ts string) bool {
```

Comment:
This does not seem to work as intended: https://play.golang.org/p/ITLMRqt30ZU
It's also a good unit test contender

Reply:
I agree this should work in isolation. In the context we are using it the members of the `DateRange` struct is always padded with 0's or 9's to a length of 14.

Reply:
An issue should be created when this gets merged about making the function more "fool proof"